package.json

```diff
@@ -2,33 +2,30 @@
   "name": "create-t3-app",
   "version": "4.0.5",
   "description": "Create web application with the t3 stack",
+  "author": "Shoubhit Dash <youremail@here.com>",
```

Comment:
@nexxeln Be sure to add your email in here as I believe GitHub actually uses the information

Reply:
shoubhit2005@gmail.com

Reply:
Done